canonical-web-and-design/www.ubuntu.com
# Clean-up and link definitions in /uasd

## Done

- Updated the /uasd per Stephan's updates

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/uasd](http://0.0.0.0:8001/uasd)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1vQNdVmpsB3iQP7SnJHxT3A8rbD5jSAzTEoYUOepR-SM/edit?ts=5cca1067#)
- See if the links to definitions make sense


## Issue / Card

Fixes #5079
